docs/android-setup.md

```diff
@@ -1,45 +1,31 @@
 # Installation
 
-In some cases, simply doing `react-native link` will correctly set up the tracker on Android.
-However, if you encounter any issues (either with the command itself or at compile-time/run-time),
-you should revert any changes made by `react-native link` and try the following steps, instead.
+Run `react-native link` to link the library then add the following to `android/build.gradle`:
 
-Add the following to your `settings.gradle`:
-
-  ```groovy
-  include ':react-native-heap'
-  project(':react-native-heap').projectDir =
-      new File(rootProject.projectDir, '../node_modules/@heap/react-native-heap/android')
-  ```
```

Comment:
Is this omission correct?

Reply:
Ah, yeah, this was required for versions below 0.60.